CONTRIBUTING.md

```diff
@@ -31,17 +31,23 @@ All developers including new distributions and plugin developers are invited to
 
 ### How to submit PR's (pull requests), patches, and bug fixes
 
-- Avoid merging master in your branch because it makes code review a lot harder.
-- If you want to keep your branch up to date with our master, it would be nicer if you could just rebase your branch instead. That would keep the history a lot cleaner.
-- Please avoid adding unrelated changes in the PR. Do a separate PR and rebase once they get merged can work really well.
- 
-If you make trivial changes, you can send them directly via a pull request. **Please make your changes in a new git branch and send your patch**, including appropriate test cases.
+Welcome new contributor!
+Follow this guide [Your first pull request](https://gitter.im/OpenRefine/OpenRefine)
 
-We want to keep the quality of the trunk at a very high level, since this is ultimately where the Stable Releases are built from after bugs are fixed. Please take the time to test your changes (including travis-ci) before sending a pull request.
+- Avoid merging master in your branch because it makes code review a lot harder. **Please make your changes in a new git branch and send your patch**, including appropriate test cases.
+- If you want to keep your branch up to date with our master, it would be nicer if you could just rebase your branch instead. That would keep the history a lot cleaner.
+- Please avoid adding unrelated changes in a PR. Use separate PR's and rebase once they get merged can work really well.
+- If you make trivial changes, you can send them directly via a PR.
 
-OpenRefine is volunteer supported. Pull Requests are reviewed and merged by volunteers. All Pull Requests will be answered, however it may take some time to get back to you. Thank you in advance for your patience.
+### Code Quality ###
```

Comment:
This probably could be relabeled as **Code Quality and Style** ?  Where we also mention Code Quality checking such as our usage of Coveralls?  or we can separate into 2 sections: Code Quality and Code Style.  Up to you guys.